### PR TITLE
Update EmailVerificationTemplate AB#10046

### PR DESCRIPTION
**Pull request by tiagograf-AOT**
# Fixes or Implements [AB#10046](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/10046)

* [x] Enhancement
* [ ] Bug
* [ ] Documentation

## Description

Update EmailVerificationTemplate.

If you are reviewing this PR, please adhere to the guidelines as [documented](https://github.com/bcgov/healthgateway/wiki/prguidance).

## Testing

* [ ] Unit Tests Updated
* [ ] Functional Tests Updated
* [x] Not Required

### Steps to Reproduce

If this is a bug, please provide details on how to reproduce the code.

### UI Changes

YES | NO

### Browsers Tested

* [ ] Chrome
* [ ] Safari
* [ ] Edge
* [ ] Firefox

Provide an explanation for any test(s) not completed.

## Notes/Additional Comments

Any additional notes or comments that may be relevant.  Include any specialized deployment steps here.

### CODE REVIEW

**Review by FuriousLlama — APPROVED** [review]

File: Apps/Database/src/Assets/docs/EmailValidationTemplate.html [hunk 1]
```diff
@@ -22,6 +22,7 @@ <h1 style = "font-size:18px;">Almost there!</h1>
                 <p>We've received a request to register your email address for a Health Gateway account.</p>
                 <p>To activate your account, please verify your email by clicking the link:</p>
                 <a style = "color:#1292c5;font-weight:600;" href = "${ActivationHost}/ValidateEmail/${InviteKey}"> Health Gateway Account Verification </a>
+                <p>This email verification link will expire in ${ExpiryHours} hours.</p>
```
Comment (leobcgov) [inline Apps/Database/src/Assets/docs/EmailValidationTemplate.html:25]: I don't agree with this naming. Ideally the same name should be used across our code, db and configuration. Using `ExpiryHours` here may be ok as we know the context , but as we need to use this name also in the WebConfig, I suggest to give it a more meaningful name such as EmailVerificationExpiryHours as `ExpiryHours` in the WebConfig doesn't say much about this context.

**Review by leobcgov — CHANGES_REQUESTED** [review]
name is not meaningful.

File: Apps/Database/src/Assets/docs/EmailValidationTemplate.html (hunk 1 above)
Comment (tiagograf-AOT) [reply Apps/Database/src/Assets/docs/EmailValidationTemplate.html:25]: The config and the variable don't have to have the same name, in this case the config is stored in seconds and the email displays it in hours, this means they will not have the same name anyway.

I'm following the standard for the other variables, eg. its InviteKey and not EmailVerificationInviteKey

Comment (leobcgov) [reply Apps/Database/src/Assets/docs/EmailValidationTemplate.html:25]: resolved per our team discussion, approved

**Review by leobcgov — APPROVED** [review]

**Pull request merged**